docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+---
+postgres:
+  image: postgres:9.5
+  environment:
+    POSTGRES_PASSWORD: mysecretpassword
+    POSTGRES_USER: postgres
+    POSTGRES_DB: flyway
+   
+  restart: always
+  ports:
+   - '5432:5432'
```

Comment:
@donaldchampion  I don't think you need to expose the Postgres port externally. Using Docker link/networking you can connect to it without exposing the port.